pkg/ebpf/tracee.go

```diff
@@ -619,6 +620,7 @@ func (t *Tracee) populateBPFMaps() error {
 	errs = append(errs, t.initTailCall(tailVfsWritev, "prog_array", "trace_ret_vfs_writev_tail"))
 	errs = append(errs, t.initTailCall(tailSendBin, "prog_array", "send_bin"))
 	errs = append(errs, t.initTailCall(tailSendBinTP, "prog_array_tp", "send_bin_tp"))
+	errs = append(errs, t.initTailCall(tailKernelWrite, "prog_array", "trace_ret_vfs_write_tail"))
```

Comment:
if i understand correctly, this should be `trace_kernel_write_tail`

Reply:
I must have mistyped it xD